JavaScript-foundations-feat-challenge_03_loops-conditionals/loops-conditionals/loops/exercise_01.js

```diff
@@ -0,0 +1,18 @@
+/*
+    1. Create a for loop that loops while index is less or equal than 10.
+    2. Each loop:
+        * create a new random number between 1 and 10 (hint: check documentation regarding Math.floor and Math.random methods)
+        * evaluate if random number is greater than 5
+        * if it is equal or greater than 5 print "Logged In!"
+        * if it is less than 5 print "Logged Out!"
+*/
+
+for (let i = 1; i <= 10; i++) {
```

Comment:
You should take in account what `i` variable is storing. Is this variable referring to any index in this case, or a number?

⚠️ FIX: rename properly `i` variable